homeassistant/components/rfxtrx/sensor.py

```diff
@@ -252,7 +252,7 @@ def __init__(self, device, device_id, entity_description, event=None):
         """Initialize the sensor."""
         super().__init__(device, device_id, event=event)
         self.entity_description = entity_description
-        self._name = f"{device.type_string} {device.id_string} {entity_description.key}"
+        self._attr_name = entity_description.key
```

Comment:
This is the default behavior when an entity description is set, and should be removable?

Reply:
default is `.name` not `.key`. I would have to duplicate the key names in the entity_description list.

Reply:
I decided to add the names. It solves some confusion on the naming anyway.